### DISCUSSION TITLE
Any news for 11 mini next update.

### DISCUSSION THREAD

**Discussion by audetra** [gopro/labs#942] [General]
I see gopro has updated for 11 mini on May 2, 2024 V2.50.  Any news for when the labs will support the latest version ?

**Comment by dnewman-gpsw** [comment]
It is coming in a day or so.

**Comment by dnewman-gpsw** [comment]
Up now.

**Reply by RikshaDriver** [reply]
What was the issue behind previous release getting pulled?

**Reply by dnewman-gpsw** [reply]
There wasn't any user facing bugs, however the build was done from the wrong branch so it wasn't exactly 2.50 + Labs.  2.50.71 is better, the correct foundation branch and the latest Labs fixes.

**Comment by audetra** [comment]
Up and running, so far everything looks good.
Thank-you.
